mapiproxy/servers/default/nspi/emsabp.c

```diff
@@ -633,6 +625,13 @@ _PUBLIC_ void *emsabp_query(TALLOC_CTX *mem_ctx, struct emsabp_context *emsabp_c
 		}
 		data = (void *) mvszA;
 		break;
+	case PT_LONG:
```

Comment:
This is needed because  PR_OBJECT_TYPE and PR_DISPLAY_TYPE are not longer hardcoded and both are of this type